pkgs/top-level/aliases.nix

```diff
@@ -934,6 +934,7 @@ mapAliases ({
   lua5_sec = throw "'lua5_sec' has been renamed to/replaced by 'luaPackages.luasec'"; # Converted to throw 2022-02-22
   lumo = throw "lumo has been removed: abandoned by upstream"; # Added 2022-04-25
   lumpy = throw "lumpy has been removed from nixpkgs, as it is stuck on python2"; # Added 2022-01-12
+  lvm2-2_02 = lvm2;
```

Comment:
That lvm2 was only used to give a lvm in the static case, I doubt anyone has been adding `lvm2-2_02` explicitly.

I'd drop it alltogether without creating an alias.

Reply:
yeah, I wasn't sure if we should add an alias. I don't really have a strong opinion on adding it or not, so if you feel like we shouldn't have it, I can drop it.

Reply:
👍